### DISCUSSION TITLE
Invalid IPs on domain list

### DISCUSSION THREAD

**Discussion by pallebone** [FiltersHeroes/KADhosts#76] [Q&A] [answered]
Hello, 

Pihole reports ips are on the list:
```
  [i] Target: https://raw.githubusercontent.com/FiltersHeroes/KADhosts/master/KADomains.txt
  [✓] Status: Retrieval successful
  [✓] Parsed 143533 exact domains and 0 ABP-style domains (ignored 4 non-domain entries)
      Sample of non-domain entries:
        - "51.210.43.4"
        - "102.130.124.5"
        - "159.65.193.7"
        - "185.34.52.4"
```

Kind regards
Peter

**Comment by krystian3w** [comment]
Maybe due works fine in Forti Guard: https://github.com/FiltersHeroes/KADhosts/issues/34#issuecomment-720718091

**Comment by hawkeye116477** [comment, marked as answer]
For Pi-hole you can use standard hosts file => https://raw.githubusercontent.com/FiltersHeroes/KADhosts/master/KADhosts.txt.

**Comment by pallebone** [comment]
> Maybe due works fine in Forti Guard: [#34 (comment)](https://github.com/FiltersHeroes/KADhosts/issues/34#issuecomment-720718091)

Hi,

I dont understand your reply. Can you help explain what you are telling me?

kind regards
Pete

**Comment by krystian3w** [comment]
List was made for Forti Guard.

**Comment by hawkeye116477** [comment]
That's right. Everything is fine, it's list not for Pi-hole.

https://www.youtube.com/watch?v=EvMx8_Ok3ck

**Comment by pallebone** [comment]
Thank you, this list works:
```
  [i] Target: https://raw.githubusercontent.com/FiltersHeroes/KADhosts/master/KADhosts.txt
  [✓] Status: Retrieval successful
  [✓] Parsed 148437 exact domains and 0 ABP-style domains (ignored 0 non-domain entries)
```
Kind regards
Peter

[a comment by an automated account is not shown]
